Chapter_2/2_10_DisturbedClosedOrbits.tex

```diff
@@ -130,47 +130,24 @@ \section{Disturbed closed orbits}\label{sec:2.10}
  \end{align*}
 
- where $\mathcal{M}$ is a $2m \times n_x + n_y$ called orbit response matrix, whose terms are
-
- \begin{align*}
-M_{ij} = \dfrac{\sqrt{\beta_i}}{2\sin\pi\nu}\sqrt{\beta_j}\cos\{ |\varphi_i - \varphi_j| - \pi\nu \}.
-\end{align*}
-
-The distorted closed orbit can be corrected at least where the BPMs are placed and we wish that the displacement produced by the correction magnets cancels out the orbit displacement produced by field errors, i.e., $\mathbf{x}_{2m} + \Delta \mathbf{x}_{2m} = 0$, then $\mathbf{x}_{2m} + \mathcal{M}\Delta\boldsymbol{\theta}_{n_x + n_y} = 0$. Therefore
+ $\mathrm{N}_{\mathrm{BPM}}$ is the number of BPMs, $\mathrm{N}_{\mathrm{CH}}$ is the number of horizontal correctors (CH) and $\mathrm{N}_{\mathrm{CV}}$ is the number of vertical correctors (CV). In this vectorial form, the orbit distortion is related to the kick variation by $\Delta \vec{u} = \mathbf{M} \Delta \vec{\theta}$, and the orbit response matrix has dimension $2 \mathrm{N}_{\mathrm{BPM}} \times \left(\mathrm{N}_{\mathrm{CH}} + \mathrm{N}_{\mathrm{CV}}\right)$.
 
-\begin{align*}
-  \Delta \boldsymbol{\theta}_n = - \mathcal{M}^{-1}\Delta \mathbf{x}_m.
-\end{align*}
-
-If $2m=n_x+n_y$ we obtain a square response matrix $\mathcal{M}$ and its inverse is obtained by the ordinary inversion. When $2m < n_x+n_y$ or $2m > n_x + n_y$ , the response matrix is rectangular. A commom method used to obtain the pseudo-inverse of the response matrix is the Singular Value Decomposition (SVD). When the system is overdetermined the SVD pseudo-inversion gives the solution for the minimum correctors strength that corrects the orbit distortion measured at BPMs. When underdetermined its only possible to find solution for the correctors that minimizes the BPMs readings.
+ Suppose that the BPMs measure the orbit distortion $\vec{u}_{\mathrm{d}}$.With the dipolar correctors it is possible to produce an orbit variation that minimizes the 2-norm of the residual orbit. From the minimization of $||\vec{u}_{\mathrm{d}} - \mathbf{M}\Delta \vec{\theta}||^2$ with respect to $\Delta\vec{\theta}$, the required kick variations are $\Delta \vec{\theta} = -\left(\mathbf{M}^{\mathsf{T}}\mathbf{M}\right)^{-1}\mathbf{M}^{\mathsf{T}}\vec{u}_{\mathrm{d}}$. Since the actual problem may be non-linear, the corrections must be calculated and applied iteratively until convergence.
 
-\begin{align*}
-  \left(\mathbf{x}_m - \Delta \mathrm{x}_m\right)^{2}_{\text{min}}=  \left(\mathbf{x}_m - \mathcal{M}\Delta \boldsymbol{\theta}_n\right)^{2}_{\text{min}}.
-\end{align*}
+ The pseudo-inverse of $\mathbf{M}^{\mathsf{T}}\mathbf{M}$ can be obtained, for example, with the method of Singular Value Decomposition (SVD). The SVD-based orbit correction is quite common in storage rings but there are other algorithms also available, depending on the application purpose.
```

Comment:
(SVD) with a suitable choice of the singular values used in the inversion procedure.